src/pages/workspace/categories/CategorySettingsPage.tsx

```diff
@@ -49,6 +54,20 @@ function CategorySettingsPage({route, policyCategories}: CategorySettingsPagePro
         Navigation.navigate(ROUTES.WORKSPACE_CATEGORY_EDIT.getRoute(route.params.policyID, policyCategory.name));
     };
 
+    const deleteCategory = () => {
+        Policy.deleteWorkspaceCategories(route.params.policyID, [route.params.categoryName]);
+        setDeleteCategoryConfirmModalVisible(false);
+        Navigation.dismissModal();
```

Comment:
We should have moved to previous screen here. This later caused https://github.com/Expensify/App/issues/43616